docs/src/development/sanitize-db.md

```diff
@@ -0,0 +1,143 @@
+
+Make sure that the sanitization only runs on the development environments and **never** on the production environment.
+Otherwise you will loose most — if not all — of the relevant data stored in your database.
+
+{{< codetabs >}}
+
+---
+title=Manually
+file=none
+highlight=false
+---
+
+Assumptions:
+
+- `staging` is the name of the database that you want to sanitize.
+- The `staging` database is an exact copy of your production database.
+- `user` is the table where all your PII is stored.
+
+1. As a safety precaution, take a [database dump](../configuration/services/mysql.md#exporting-data). Run `platform db:dump -e staging` to get a database dump of your staging environment,
+1. Connect to the `staging` database with `platform sql -e staging`,
+1. Display all fields from your `user` database with `SELECT * FROM users;`.
+   Which could result in something along the lines of:
+
+   ```sql
+   MariaDB [main]> SELECT * FROM users;
+   +----+------------+---------------+---------------------------+---------------+
+   | ID | first_name | last_name     | user_email                | display_name  |
+   +----+------------+---------------+---------------------------+---------------+
+   |  1 | admin      | admin         | admin@yourcompany.com     | admin         |
+   |  2 | john       | doe           | john.doe@gmail.com        | john          |
+   |  3 | jane       | doe           | janedoe@ymail.com         | jane          |
+   +----+------------+---------------+---------------------------+---------------+
+   3 rows in set (0.00 sec)
+   ```
+
+1. Change the fields where PII Data is contained with [`UPDATE` Queries](https://dev.mysql.com/doc/refman/8.0/en/update.html).
+   For example, changing the first name of an user, when the email address is not using the company domain, would result in the following query:
+
+   ```sql
+   UPDATE user
+   SET first_name='redacted'
+   WHERE email NOT LIKE '%@yourcompany%'
+   ```
+
+   Adapt and run that query for all fields that need to be sanitized.
+   If you modified fields that shouldn't have been altered, [you can restore them from the previously taken dump](../administration/backup-and-restore.md#restore).
+
+   You can create a script to automate the sanitization process to be able to run it in [your deployment hook](../user_guide/reference/platform-app-yaml.html#hooks):
+
+    ```yaml
+    deploy: |
+      cd /app/public
+      if [ $PLATFORM_ENVIRONMENT = "main" ]; then
+        # Do whatever you want on the production site.
+      else
+        # The sanitization of the database should happen here (since it's non-production)
+      fi
+    ```
+
+<--->
+
+---
+title=When using Drupal
+file=none
+highlight=false
+---
+
+1. Take a [database dump](../configuration/services/mysql/_index.md#exporting-data). In this guide production data won't be altered, consider it merely as a safety precaution. Run `platform db:dump -e staging` to get a database dump of your staging environment,
```

Comment:
The comments above about semantic line breaks and the last part of the last sentence apply here, too.

Also, I'd change
> In this guide production data won't be altered, consider it merely as a safety precaution.
to something like
> You won't alter your production data. This is just a safety precaution.


The bit about it being a safety precaution is good and probably should be added above, too.
Also, if it's true for both paths, you could consider taking it out of the codetabs and having it only once.